node-task-pkg/index.js

```diff
@@ -0,0 +1,51 @@
+const util = require("util");
+const { exec } = require("child_process");
+const execProm = util.promisify(exec);
+
+const os = require("os");
+const plat = os.platform();
+
+exports.getProcList = async function (username) {
+    let result = {stdout:"",stderr:""};
+    try{
+        if (plat == "win32"){
+            result = await execProm("C:/Windows/System32/tasklist.exe /FO CSV")
+            result.stdout = result.stdout.trim().split("\r\n")
```

Comment:
I think we can use array method map here instead of for loop. It will be better to read